src/navigation/drawer.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import{createDrawerNavigator} from 'react-navigation-drawer';
 import {createAppContainer} from 'react-navigation';
-import Home from'../views/screens/home';
-import ArtworkDetails from '../views/screens/artworkDetails';
-import Cart from '../views/screens/artworkDetails';
-import Checkout from '../views/screens/checkOut'
-import Logout from '../views/screens/logout'
-import Profile from '../views/screens/profile';
+import Home from'../views/components/home';
```

Comment:
If it renders the whole screen, move that to the views directory, and if it renders a component that is reusable throughout the app, then keep it inside the components directory.